tests/lib/managers/folders-test.js

```diff
@@ -276,4 +276,21 @@ describe('Folders', function() {
 		});
 	});
 
+	describe('getTrashedItems()', function() {
+
+		it('should make GET call to get trashed items', function() {
+
+			sandbox.stub(boxClientFake, 'defaultResponseHandler');
+			sandbox.mock(boxClientFake).expects('get').withArgs('/folders/trash/items', testParamsWithQs);
+			folders.getTrashedItems(testQS);
+		});
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
+
+			sandbox.stub(boxClientFake, 'defaultResponseHandler').withArgs(done).returns(done);
```

Comment:
This should be a mock, so that the test will fail with a helpful error message if defaultResponseHandler gets the wrong argument.